profiler/profile.go

```diff
@@ -319,15 +319,16 @@ type pprofileDeltaProfiler struct {
 // Otherwise, deltas will be computed for every value.
 func newDeltaProfiler(cfg *config, v ...pprofutils.ValueType) deltaProfiler {
 	switch cfg.deltaMethod {
-	case "fastdelta":
-		return newFastDeltaProfiler(v...)
+	// TODO: slowdelta and comparing implementation can be removed after 2022-04-11.
```

Comment:
What's the significance of 2022-04-11? Do you mean 2023-04-11, and is there a release scheduled for that date?

Reply:
Yes, I meant `2023-04-11`. I picked "3 months from now" to give customers ~3 releases where they can still get the old behavior back. Arguably we could also do this a bit earlier and align with the release schedule, but I figured that's good enough. Fixed the year in f4ba9987849f895bbb108946d6576cb183f73344.